packages/grafana-ui/src/components/CallToActionCard/CallToActionCard.story.tsx

```diff
@@ -3,7 +3,7 @@ import { storiesOf } from '@storybook/react';
 import { renderComponentWithTheme } from '../../utils/storybook/withTheme';
 import { CallToActionCard } from './CallToActionCard';
 import { select, text } from '@storybook/addon-knobs';
-import { Button } from '../Button/Button';
+import { Button } from '../Forms/Legacy/Button/Button';
```

Comment:
Do we need the legacy Button? what are the api differences?

Reply:
No, I don't think so. The only difference is is variants. Currently looking at how much we can move into the new Button.

Reply:
There are components like the FullWidthButtonContainer that needs to be taken into consideration too.